src/frontend/screens/SignUpScreen/SignUpScreen.js

```diff
@@ -1,13 +1,135 @@
-import React from "react";
+import React, { useState } from "react";
+import { Link, useNavigate } from "react-router-dom";
+import { AUTH_TOKEN, USERNAME } from "../../constants/authConstants";
+import { useAuth } from "../../contexts";
+import { signUpService } from "../../services/auth-services/signUpService";
+import "./SignUpScreen.css";
 
 const SignUpScreen = () => {
+  const [user, setUser] = useState({
+    firstName: "",
+    lastName: "",
+    email: "",
+    password: "",
+    confirmPassword: "",
+  });
+
+  const { setAuth } = useAuth();
+  const navigate = useNavigate();
+
+  const signUpHandler = async (user) => {
+    const res = await signUpService(user);
+    const { createdUser, encodedToken } = res;
+    if (encodedToken !== undefined) {
+      localStorage.setItem(AUTH_TOKEN, encodedToken);
+      localStorage.setItem(USERNAME, createdUser.firstName);
+      setAuth((auth) => ({
+        ...auth,
+        status: true,
+        token: encodedToken,
+        userName: createdUser.firstName,
+      }));
+    }
+    navigate("/");
+  };
+
   return (
-    <div
-      className="main-content"
-      style={{ color: "white", textAlign: "center", marginTop: "10rem" }}
+    <form
```

Comment:
Remove inline style if possible.

Reply:
Thanks for noticing this. Forgot to remove it.

Reply:
I don't see any inline styling here. It is removed already.